rocAL/CMakeLists.txt

```diff
@@ -297,10 +297,12 @@ if(${BUILD_ROCAL})
         target_compile_definitions(${PROJECT_NAME} PUBLIC ENABLE_HIP=1)
         target_compile_definitions(${PROJECT_NAME} PUBLIC ENABLE_OPENCL=0)
         if(ROCDECODE_FOUND)
-            target_compile_definitions(${PROJECT_NAME} PUBLIC ENABLE_ROCDECODE=1)
-            message("-- ${White}rocAL built with rocDecode video decode capabilities${ColourReset}")
-        else()
-            target_compile_definitions(${PROJECT_NAME} PUBLIC ENABLE_ROCDECODE=0)
+            if(${ROCDECODE_VER_MAJOR} VERSION_GREATER_EQUAL "0" AND ${ROCDECODE_VER_MINOR} VERSION_GREATER_EQUAL "10" AND ${ROCDECODE_VER_MICRO} VERSION_GREATER_EQUAL "0")
```

Comment:
Please use this function instead: https://github.com/ROCm/rocDecode/blob/1f53dedd4a7562a353e1c52c605e530e082133b8/api/rocdecode_version.h#L51